docs/excel/custom-functions-batching.md

```diff
@@ -0,0 +1,214 @@
+---
+ms.date: 04/22/2019
+description: "Batch custom functions together to reduce network calls to a remote service."
+title: Batching custom function calls for a remote service
+localization_priority: Priority
+---
+
+# Batching custom function calls for a remote service
+
+If your custom functions call a remote service you can use a batching pattern to reduce the number of network calls to the remote service. To reduce network round trips you batch all the calls into a single call to the web service. This is ideal when the spreadsheet is recalculated. For example, if someone used your custom function in 100 cells in a spreadsheet, and then recalculated the spreadsheet, your custom function would run 100 times and make 100 network calls. By using a batching pattern, the calls can be combined to make all 100 calculations in a single network call.
```

Comment:
David, per our style guide, you need to avoid first person plural. I noticed in this topic, you use "we" a lot. I offered suggestions in a few places but I think you should go through the whole article and fix it yourself so I don't inadvertantly change the meaning.
[Style Guide: Avoid first-person plural](https://worldready.cloudapp.net/Styleguide/Read?id=2700&topicid=25524)